tests/utils/iotrace.py

```diff
@@ -120,132 +184,282 @@ def get_sort_key(element):
 
-    def get_trace_summary(self, trace_path: str) -> str:
-        '''
+    @staticmethod
+    def get_trace_summary(trace_path: str,
+                          shortcut: bool = False) -> str:
+        """
         Get trace summary of given trace path
 
         :param trace_path: trace path
-        :return: Summary of trace in JSON format
+        :param shortcut: Use shorter command
+        :type trace_path: str
+        :type shortcut: bool
+        :return: Summary of trace
         :raises Exception: if summary is invalid
-        '''
-        output = TestRun.executor.run(
-            f'iotrace --trace-management --get-trace-summary -p {trace_path}')
-        if (output.stdout == ""):
+        """
+        command = 'iotrace' + (' -M' if shortcut else ' --trace-management')
+        command += ' -G' if shortcut else ' --get-trace-summary'
+        command += (' -p ' if shortcut else ' --path ') + f'{trace_path}'
+        output = TestRun.executor.run(command)
+        if output.stdout == "":
             raise Exception("Invalid summary")
 
-        return output.stdout
+        return parse_json(output.stdout)[0]
 
-    def get_lba_histogram(self, trace_path: str, bucket_size=0,
-                          subrange_start=0, subrange_end=0) -> str:
-        '''
+    @staticmethod
+    def get_lba_histogram(trace_path: str,
+                          bucket_size: Size = Size(0, Unit.Byte),
+                          subrange_start: int = 0,
+                          subrange_end: int = 0,
+                          shortcut: bool = False) -> str:
+        """
         Get lba histogram of given trace path
 
         :param trace_path: trace path
         :param bucket_size: bucket size
         :param subrange_start: subrange start
         :param subrange_end: subrange end
-        :return: LBA histogram in JSON format
+        :param shortcut: Use shorter command
+        :type trace_path: str
+        :type bucket_size: Size
+        :type subrange_start: int
+        :type subrange_end: int
+        :type shortcut: bool
+        :return: LBA histogram
         :raises Exception: if histogram is invalid
-        '''
-        output = TestRun.executor.run(
-            f'iotrace --trace-parser --lba-histogram -p {trace_path} -b {bucket_size} -s {subrange_start} -e {subrange_end}')
-        if (output.stdout == ""):
+        """
+        bucket_size_range = range(1, 4294967296)
+        subrange_range = range(1, 9223372036854775808)
+        if subrange_start and subrange_end:
+            if subrange_start > subrange_end:
+                subrange_start, subrange_end = subrange_end, subrange_start
+
+        command = 'iotrace' + (' -P' if shortcut else ' --trace-parser')
+        command += ' -B' if shortcut else ' --lba-histogram'
+        command += (' -p ' if shortcut else ' --path ') + f'{trace_path}'
+        if bucket_size is not None \
+                and int(bucket_size.get_value(Unit.Byte)) in bucket_size_range:
```

Comment:
I don't think it's the best idea to just silently discard arguments that are not in range. What I'd expect is that if I don't specify the argument I get the default, but if I specify wrong argument then raise exception.